data.sql

```diff
@@ -16,3 +16,34 @@ INSERT INTO animals
 (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
 ('Devimon', '2015-05-12', 5, true, 11);
 
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Charmander', '2020-02-08', 0, false, 11);
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Plantmon', '2021-11-15', 2, false, 5.7);
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Squirtle', '1993-04-2', 3, false, 12.13);
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Angemon', '2005-06-12', 45, false, 1 );
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Boarmon', '2005-06-7', 20.4, true, 7 );
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Blossom', '1998-10-13', 17, true, 3 );
+
+INSERT INTO animals 
+(name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+('Ditto', '2022-05-14', 22, true, 4 );
```

Comment:
- Good job done so far, but you seem to have switched values for the weights and escape attempt attributes, please make sure you properly and carefully input data as given in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md) 👍 👍